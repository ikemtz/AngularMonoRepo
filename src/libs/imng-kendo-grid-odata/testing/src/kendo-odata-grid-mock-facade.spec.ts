import { createODataGridMockFacade } from './kendo-odata-grid-mock-facade';

describe('Mock OData Facade Creator', () => {
  it('createODataGridMockFacade() should work with undefined', async () => {
    const result = createODataGridMockFacade();
    expect(result).toMatchSnapshot();
  });
  it('createODataGridMockFacade() should work', async () => {
    const result = createODataGridMockFacade({});
    expect(result).toMatchSnapshot();
  });
});
