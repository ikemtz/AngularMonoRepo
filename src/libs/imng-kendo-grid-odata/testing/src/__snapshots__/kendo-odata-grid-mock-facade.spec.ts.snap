// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mock OData Facade Creator createODataGridMockFacade() should work 1`] = `
Object {
  "gridODataState$": Observable {
    "_isScalar": false,
    "_subscribe": [Function],
  },
  "gridPagerSettings$": Observable {
    "_isScalar": false,
    "_subscribe": [Function],
  },
  "loadEntities": [MockFunction],
  "loading$": Observable {
    "_isScalar": false,
    "_subscribe": [Function],
  },
  "reloadEntities": [MockFunction],
}
`;

exports[`Mock OData Facade Creator createODataGridMockFacade() should work with undefined 1`] = `
ODataGridMockFacade {
  "gridData$": Observable {
    "_isScalar": false,
    "_subscribe": [Function],
  },
  "gridODataState$": Observable {
    "_isScalar": false,
    "_subscribe": [Function],
  },
  "gridPagerSettings$": Observable {
    "_isScalar": false,
    "_subscribe": [Function],
  },
  "loadEntities": [MockFunction],
  "loading$": Observable {
    "_isScalar": false,
    "_subscribe": [Function],
  },
  "reloadEntities": [MockFunction],
}
`;
